### ISSUE TITLE
Pushing to Github through SourceTree error: remote: Support for password authentication was removed on August 13, 2021

### ISSUE THREAD

**Issue by ChangruHenryQian** [nus-cs2103-AY2324S1/forum#17]
I am following the instructions on W2.4e Tools → Git and GitHub → push: Uploading data to other repos to practice pushing to repo under my Github account. Credential Helper Selector keeps poping up and asks me to select a credential helper. I selected <no helper> and was asked to log in to my Github account. I was getting the above error after that. May I know how to fix this?
Error message:
```
git -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks push -v --tags myfork master:master
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ChangruHenryQian/samplerepo-things.git/'

Pushing to https://github.com/ChangruHenryQian/samplerepo-things.git
Completed with errors, see above.
```

**Comment by lamchenghou** [comment]
I think you need to set up a Personal Access Token in GitHub, and use that as the passkey instead.

**Comment by KamJiaYue** [comment]
https://youtu.be/ytSoabxSQ6E?feature=shared

can try follow this video maybe?

**Comment by damithc** [comment]
@ChangruHenryQian If you are using Sourcetree, you can refer to [this guide](https://se-education.org/guides/tutorials/sourcetree.html#:~:text=Connecting%20Sourcetree%20to%20Your%20GitHub%20Account).

**Comment by ChangruHenryQian** [comment]
> @ChangruHenryQian If you are using Sourcetree, you can refer to [this guide](https://se-education.org/guides/tutorials/sourcetree.html#:~:text=Connecting%20Sourcetree%20to%20Your%20GitHub%20Account).

@damithc Thank you for your reply Professor.

I am using Win11. 
Option 1: I followed the entire instruction and was still asked to select a credential helper. I selected `wincre` and everything works fine for now. If things go wrong in the future, I will post updates.
Option 2: Doesn't work because `Basic` is not in the list I can select from in the Authentication section in SourceTree.(Step 5)
Option 3: Checked Windows Credential Manager and didn't find anything related to Github.
